CONTRIBUTING.md

```diff
@@ -61,31 +61,91 @@ You can also clean up generated files:
+
+We enforce constraints ensuring that all pull requests will result in a merge commit to the master branch which satisfies Conventional Commit (according to the [Conventional Commit 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/) spec), by ensuring that either the pull request's title and/or commit messages satisfy the conditions of the [release workflow outlined above](#release-workflow).
+
+These constraints require that each pull request have a title formatted as follows:
+
+```
+<type>: <description>
+```
+
+#### Description
+
+The `description` should be a short (less than ~60 characters) summary of changes introduced in the release. This summary will be included in the CHANGELOG (see the [releases page](https://github.com/cultureamp/kaizen-design-system/releases)), along with the commit type, a link to the pull request which triggered the release, and a detailed summary of any breaking changes.
+
+Note that since the `description` will be included in the CHANGELOG — and may be the only summary of the changes that your colleagues will encounter — it is helpful for you to include a concise summary of _how the package is different_ following the release, written in the [imperative mood](https://chris.beams.io/posts/git-commit/#imperative). For example `fix: Address accessibility bug in the Gizmo component`.
+
+#### Type
 
-For example, a PR title of `feat: change API for X component` with a squashed commit that contains BREAKING CHANGE will create a **major** release, bumping the version from 16.2.1 to 17.0.0.
+The commit `type` is used to indicate the type (i.e. patch, minor, major in [Semantic Versioning](#semantic-versioning)) of change that was included in the release, as well as the context in which that change was made.
 
-For another example, a PR title of `feat: add a new Toggle component` will create a **minor** release, bumping the version from 16.0.0 to 16.1.0.
+There are a number of possible commit types (e.g. `chore`, `docs`, `refactor` etc. — for a complete list, refer to the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type) docs), but two have special meaning for the purpose of versioning package releases:
 
-Finally, a PR title of `fix: fix the focus styling on our button component` will create a **patch** release, bumping the version from 16.0.0 to 16.0.1.
+- `fix` — indicates that a change addressed a bug or security concern in the published code, but otherwise had no consequence for the released package's features or API (corresponding to a PATCH version)
 
-## Using your newly released component changes
+- `feat` — indicates that a change added something new to the released package's API or features without affecting existing functionality (corresponding to a MINOR version)
```

Comment:
Can we include a concrete example of `docs`? That's come up a few times and its absence here is noticed. That is, people look for how to update `docs` and it's not listed.